datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/jmx.yaml

```diff
@@ -10,6 +10,11 @@
   value:
     type: integer
 
+- name: jmx_url
+  description: JMX URL to connect to. Can be used instead of host/port configs.
+  value:
```

Comment:
Shall we mark this as hidden since it's not needed for most integrations ?

Reply:
Is still a valid alternative, right?

Reply:
Yes, for Jboss Wildfly, because it uses another protocol.

Reply:
But I'm hiding it by default because most of the integrations only need host/port.